pubnub/models/consumer/history.py

```diff
@@ -65,7 +65,6 @@ def __str__(self):
     @classmethod
     def from_json(cls, json_input, include_message_actions=False, start_timetoken=None, end_timetoken=None):
         channels = {}
-        print(json_input['channels'])
```

Comment:
There is no simple possibility to wrap it around with `log_verbosity` config option. So it's easier to just remove this debug